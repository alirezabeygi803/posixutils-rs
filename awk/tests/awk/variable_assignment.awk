BEGIN {
    a = 1
    print a
    a = "string"
    print a
    b = "other string"
    print b
    print a
}