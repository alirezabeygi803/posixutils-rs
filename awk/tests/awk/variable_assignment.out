1
string
other string
string
